### PR TITLE
Remove .DS_Store

### PR DESCRIPTION
**Pull request by 38elements**
Since `.DS_Store` is unnecessary, this removed it. This added `.DS_Store` to `.gitignore`.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -68,3 +68,5 @@ coverage/
 # favicons - we want to ignore any downloaded .ico files but keep the directory that stores them.
 resources/favicons/*
 !resources/favicons/.gitkeep
+
```
Comment (vidorteg) [inline .gitignore:71]: nit: remove empty line

**Review by vidorteg — APPROVED** [review]
Thanks for the contribution @38elements, LGTM minus a small nit.

File: .gitignore (hunk 1 above)
Comment (38elements) [reply .gitignore]: Thank you for pointing out. I removed the line.

**Comment by codepo8** [comment]
Sorry :)

**Comment by codepo8** [comment]
Time to add to the gitignore

**Comment by bmihvkkvbk-spec** [comment]
https://github.com/microsoft/vscode-edge-devtools/issues/3963#issuecomment-3690034471

**Pull request merged**